app/_layouts/default.html

```diff
@@ -29,6 +29,16 @@
 
     {% include_cached footer.html %} {% include_cached anchor_links.html %}
 
+    {% if jekyll.environment == "production" %}
+      <script>
```

Comment:
should we place this script after the `/assets/app.js` is loaded so we know that it executes after the library was loaded?

Reply:
It has to be placed before to set `writeKey` before the library loads as there are some on-load side effects